packages/esm-patient-chart-app/src/patient-chart/chart-review/dashboard-view.component.tsx

```diff
@@ -82,14 +67,19 @@ export function DashboardView({ dashboard, patientUuid, patient }: DashboardView
     <>
       <ExtensionSlot state={state} name="top-of-all-patient-dashboards-slot" />
       {!dashboard.hideDashboardTitle && resolvedTitle && <h1 className={styles.dashboardTitle}>{resolvedTitle}</h1>}
-      <ExtensionSlot
-        key={dashboard.slot}
-        name={dashboard.slot}
-        className={styles.dashboard}
-        style={{ gridTemplateColumns }}
-      >
-        <Extension state={state}>{wrapItem}</Extension>
-      </ExtensionSlot>
+      <div className={styles.dashboardContainer}>
+        <ExtensionSlot key={dashboard.slot} name={dashboard.slot} className={styles.dashboard}>
+          {(extension) => {
+            const { fullWidth = false } = widgetMetas[getExtensionNameFromId(extension.id)];
```

Comment:
Is there a reason to call `getExtensionNameFromId()` instead of using the `extension.name` property?